### DISCUSSION TITLE
Dual boot windows linux

### DISCUSSION THREAD

**Discussion by bern6** [mytourbook/mytourbook#1168] [Q&A]
Hello,

I would like to install Windows 10, if I just rename .mytourbook(linux) to Mytourbook(windows), would it work?

Thanks

**Comment by wolfgang-ch** [comment]
> would it work?

Mostly, except file pathnames which are saved, do not work, e.g. filepath names are saved for imported files or photos

I copy my Win data to Linux when I need to test it on Linux

**Comment by bern6** [comment]
The important thing was my linux data, for the rest no problem.

Thanks
